modules/shapefile/src/lib/parsers/parse-shapefile.js

```diff
@@ -46,7 +66,7 @@ export async function* parseShapefileInBatches(asyncIterator, options, context)
       encoding: cpg,
       prj,
       shx,
-      header: shapeHeader,
+      header: {shp: shapeHeader, dbf: dbfHeader},
```

Comment:
Should I avoid changing this schema? Alternatively, I could just merge the two headers since they don't have overlapping fields.

Reply:
For now I undid this change. So the `dbfHeader` isn't exposed to the user at all and we can change this in the future.